scripts/buildRegistry.mjs

```diff
@@ -0,0 +1,132 @@
+import fs from 'fs';
+import glob from 'glob';
+import * as recast from 'recast';
+import {parse} from '@babel/parser';
+import path from 'path';
+import postcss from 'postcss';
+
+const publicUrl = process.env.REGISTRY_URL || 'http://localhost:8081';
+
+fs.rmSync('starters/tailwind/registry', {recursive: true, force: true});
+fs.mkdirSync('starters/tailwind/registry');
+
+for (let file of glob.sync('starters/tailwind/src/*.{ts,tsx}')) {
+  let name = path.basename(file, path.extname(file));
+  let {dependencies, registryDependencies, content} = analyzeDeps(file, 'tailwind');
+  let type = name === 'utils' ? 'registry:lib' : 'registry:ui';
+  let item = {
+    $schema: 'https://ui.shadcn.com/schema/registry-item.json',
```

Comment:
What is this? are we creating our own shadcn theme? where does it go in regards to our docs? or how does this work?